src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.test.tsx

```diff
@@ -84,4 +84,9 @@ describe('<DefaultTranscriptListItem />', () => {
 
     expect(wrapper.exists(TranscriptsListItemInfo)).toBe(true);
   });
+
+  it('displays selected transcript', () => {
+    wrapper = renderComponent({ isDefault: true });
+    expect(wrapper.find('.defaultTranscriptLabel')).toHaveLength(1);
```

Comment:
Let's make this test slightly more interesting and make sure that the text content of `.defaultTranscriptLabel` is`Selected`.

Reply:
Does this mean we need to wrap text inside the div or is there another way in enzyme?

Reply:
MANE Plus is here for 102, but we don't have a description for it yet.
We do have a description for MANE in OLS: https://www.ebi.ac.uk/ols/ontologies/ensemblglossary/terms?iri=http%3A%2F%2Fensembl.org%2Fglossary%2FENSGLOSSARY_0000365.
To not delay things should we 1) Create a JIRA to remind us to add MANE Plus when we get the data 2) Update the hardcoded description to that in the Glossary and create a JIRA to use OLS in the future.
Note there is potential for other attributes to be used here so we will eventually need to pick these up dynamically. How - still discussing that with HAVANA but CDM probably next step

Reply:
OK. But even if we add Mane Select and hardcode the description, we wont be able to make use of it until Thoas give us those. So yes, good idea to create JIRAs. I will do it.

Reply:
> Does this mean we need to wrap text inside the div or is there another way in enzyme to check this?

Given the following markup (from DefaultTranscriptListItem.tsx):

```
          <div className={styles.defaultTranscriptLabel}>
            {defaultTranscriptLabelMap[canonicalType]?.label}
            <QuestionButton
              helpText={defaultTranscriptLabelMap[canonicalType]?.helpText}
              styleOption={QuestionButtonOption.INLINE}
            />
          </div>
```

you can do `expect(wrapper.find('.defaultTranscriptLabel').text()).toBe('Select');`